src/VerifyDuoController.js

```diff
@@ -90,14 +81,17 @@ function (Okta, Duo, Q, CookieUtil, FactorUtil, FormController, Enums, FormType,
         // jquery decides the Content-Type instead of it being a JSON type). Enroll/Verify DUO
         // are the only two places where we actually do this.
         // NOTE - If we ever decide to change this, we should test this very carefully.
+        var rememberDevice = this.get('rememberDevice');
         return Q($.post(url, data))
         .then(function () {
           return self.doTransaction(function(transaction) {
+            if (rememberDevice) {
+              transaction.data._links.next.href += '?rememberDevice=true';
+            }
```

Comment:
Yeah, we don't access links and we should not know the next available links, the Auth SDK handles it. Let's change this, I can help you with this.